### DISCUSSION TITLE
Introducing Inspira UI: A New Open-Source UI Library for Vue and Nuxt! 🚀

### DISCUSSION THREAD

**Discussion by rahul-vashishtha** [unovue/inspira-ui#1] [Announcements]
Hey everyone! 👋

I’m excited to introduce **Inspira UI**, a fresh, open-source UI library designed specifically for Vue and Nuxt developers! 🌟

### 🌟 What is Inspira UI?

**Inspira UI** is an open-source UI library designed for **Vue** and **Nuxt** developers. It’s built with **shadcn-vue**, **@vueuse/motion**, and **TailwindCSS** to provide a collection of reusable and customizable components that make building beautiful, responsive interfaces easier and faster.

Inspired by Aceternity UI, Inspira UI brings a fresh, flexible toolkit to the Vue and Nuxt ecosystems, and I'm constantly working to add more components and features to enhance the library.

### 🚀 What’s Inside?

- A growing collection of components, including hover effects, utility components, and more.
- Components designed to be highly customizable and easy to integrate into your projects.
- Ongoing development to add even more components from Aceternity UI and new, unique elements.

### 💡 Why Use Inspira UI?

- **Built for Vue and Nuxt**: Perfectly compatible with both frameworks, leveraging their modern features.
- **Customizable and Community-Driven**: Easily adaptable to any design style, and open to community contributions.
- **Open Source**: Your feedback, suggestions, and contributions are welcome to help shape the future of Inspira UI!

### 🔧 How to Get Started?

- **Website:** [inspira-ui.com](https://inspira-ui.com)
- **GitHub Repository:** [Inspira UI GitHub](https://github.com/rahulv-official/inspira-ui)

### 💬 Let’s Discuss!

I’d love to hear your feedback, ideas, and suggestions! This discussion is open to everyone who wants to contribute, share thoughts, or ask questions. Let’s build something amazing together!

Feel free to join in, whether you're interested in contributing code, suggesting new components, or just sharing your experience with Inspira UI. Every bit of feedback helps!

Looking forward to collaborating with you all! 🚀

**Comment by Arshia-123** [comment]
yay

**Comment by leoMirandaa** [comment]
Awesome 👏 !

**Comment by craigrileyuk** [comment]
@rahul-vashishtha I'm working on doing the ImagesSlider component for the project. Should have a pull request for you to review later this week.

**Reply by rahul-vashishtha** [reply]
Cool, will wait for the PR. 
Thanks.

**Comment by llds66** [comment]
cool

**Comment by nathan-de-pachtere** [comment]
love it
